src/Kunstmaan/NodeSearchBundle/Configuration/NodePagesConfiguration.php

```diff
@@ -615,27 +590,17 @@ protected function renderCustomSearchView(
      * @param EngineInterface       $renderer
      *
      * @return string
+     *
+     * @deprecated since 5.1. Use "Kunstmaan\NodeSearchBundle\Helper\SearchRenderHelper::renderDefaultSearchView" instead.
      */
     protected function renderDefaultSearchView(
         NodeTranslation $nodeTranslation,
         HasPagePartsInterface $page,
         EngineInterface $renderer
     ) {
-        $pageparts = $this->indexablePagePartsService->getIndexablePageParts($page);
-        $view      = 'KunstmaanNodeSearchBundle:PagePart:view.html.twig';
-        $content   = $this->removeHtml(
-            $renderer->render(
-                $view,
-                array(
-                    'locale'    => $nodeTranslation->getLang(),
-                    'page'      => $page,
-                    'pageparts' => $pageparts,
-                    'indexMode' => true
-                )
-            )
-        );
+        @trigger_error(sprintf('The "%s" method is deprecated in NodeSearchBundle 5.1 and will be removed in NodeSearchBundle 6.0.', __METHOD__), E_USER_DEPRECATED);
```

Comment:
See comment above about including the possible solution in the deprecation message